crates/nu-command/tests/commands/format.rs

```diff
@@ -80,7 +80,7 @@ fn format_filesize_works_with_nonempty_files() {
 
             let actual = nu!(
                 cwd: dirs.test(),
-                "ls sample.toml | format filesize size B | get size | first"
+                "ls sample.toml | format filesize B size | get size | first"
```

Comment:
Does this mean that we'll be able to pass multiple columns?

Reply:
Yeah, like other `str` commands